nise/__main__.py

```diff
@@ -543,6 +543,15 @@ def _validate_oci_arguments(parser, options):
         (ParserError): If combination is invalid.
 
     """
+    # is_args_valid = False
+    # bucket_name = options.get("oci_bucket_name")
+
+    # if bucket_name is None:
+    #     msg = "{} must be supplied."
+    #     msg = msg.format("--oci-bucket-name")
+    #     parser.error(msg)
+    # else:
+    # is_args_valid = True
```

Comment:
Suggesiton (blocking): This validation should check if `OCI_CONFIG_FILE` is set when the `--oci-bucket-name` option is used.